blueman/main/Adapter.py

```diff
@@ -18,16 +18,14 @@
 from gi.repository import Pango
 
 
-if TYPE_CHECKING:
-    from typing_extensions import TypedDict
-
-    class Tab(TypedDict):
-        grid: Gtk.Grid
-        hidden_radio: Gtk.RadioButton
-        always_radio: Gtk.RadioButton
-        temparary_radio: Gtk.RadioButton
-        visible: bool
-        label: Gtk.Label
+class Tab(TypedDict):
```

Comment:
Makes sense. Out `TypedDict`s are from a time where we still supported Python < 3.8.